libs/blocks/table/table.js

```diff
@@ -271,6 +271,13 @@ function handleTitleText(cell) {
     nodeToInsert = titleRowSpan;
   }
 
+  const blockquote = nodeToInsert.querySelector('blockquote');
+  if (blockquote) {
+    const quoteReplacement = createTag('p', { class: 'blockquote' });
+    quoteReplacement.innerHTML = blockquote.innerHTML;
```

Comment:
We usually avoid any innerHTML copying, these can trigger security alerts and lead to event listeners being lost. Few alternatives to do this, one would be
```suggestion
    while (blockquote.firstChild) quoteReplacement.appendChild(blockquote.firstChild);
```